pegasus/helpers/form_helpers.rb

```diff
@@ -135,7 +135,15 @@ def insert_form(kind, data, options={})
     updated_ip: request.ip,
   }
   row[:user_id] = dashboard_user[:id] if dashboard_user
-  row[:id] = DB[:forms].insert(row)
+
+  if kind == "HocSignup2015"
+    update_row = DB[:forms].where(email: row[:email], kind: kind, name: row[:name]).update(row)
```

Comment:
For HocSignup2015 it gets called here: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/routes/form_routes.rb#L18 which doesn't seem to use id anywhere. Should I still do the extra query for the id?